src/uwtools/cli.py

```diff
@@ -425,6 +429,114 @@ def _dispatch_fv3(args: Args) -> bool:
     )
 
 
+# Mode mpas
+
+
+def _add_subparser_mpas(subparsers: Subparsers) -> ModeChecks:
+    """
+    Subparser for mode: mpas
+
+    :param subparsers: Parent parser's subparsers, to add this subparser to.
+    """
+    parser = _add_subparser(subparsers, STR.mpas, "Execute MPAS tasks")
+    _basic_setup(parser)
+    subparsers = _add_subparsers(parser, STR.action, STR.task.upper())
+    return {
+        task: _add_subparser_mpas_task(subparsers, task, helpmsg)
+        for task, helpmsg in uwtools.api.mpas.tasks().items()
+    }
+
+
+def _add_subparser_mpas_task(subparsers: Subparsers, task: str, helpmsg: str) -> ActionChecks:
+    """
+    Subparser for mode: mpas <task>
+
+    :param subparsers: Parent parser's subparsers, to add this subparser to.
+    :param task: The task to add a subparser for.
+    :param helpmsg: Help message for task.
+    """
+    parser = _add_subparser(subparsers, task, helpmsg.rstrip("."))
+    required = parser.add_argument_group(TITLE_REQ_ARG)
+    _add_arg_config_file(group=required, required=True)
```

Comment:
The config file can also be provided on `stdin`, so `--config-file` should be optional. This should be the case for the existing drivers, too.